velox/functions/sparksql/DecimalUtil.h

```diff
@@ -109,6 +109,21 @@ class DecimalUtil {
     return value;
   }
 
+  template <typename A, typename B>
+  inline static int32_t
+  minLeadingZeros(const A& a, const B& b, uint8_t aScale, uint8_t bScale) {
```

Comment:
Would you document this function? Let's also add a test.

Reply:
Updated.